designs/Verify-Verbosity.md

```diff
@@ -0,0 +1,240 @@
+
+The provided `--verbosity` value will include to NuGet log messages with the following levels:
+
+​             | `q[uiet]` | `m[inimal]` | `n[ormal]` | `d[etails]` | `diag[nostic]`
+------------- | --------- | ----------- | ---------- | ----------- | --------------
+`Error`       | ✔️        | ✔️         | ✔️         | ✔️         | ✔️   
+`Warning`     | ✔️        | ✔️         | ✔️         | ✔️         | ✔️   
+`Minimal`     | ❌        | ✔️         | ✔️         | ✔️         | ✔️   
+`Information` | ❌        | ❌         | ✔️         | ✔️         | ✔️   
+`Verbose`     | ❌        | ❌         | ❌         | ✔️         | ✔️   
+`Debug`       | ❌        | ❌         | ❌         | ✔️         | ✔️   
+
+Note that MSBuild itself has the following mapping for it's own "log levels"
+([source](https://docs.microsoft.com/en-us/visualstudio/msbuild/obtaining-build-logs-with-msbuild?view=vs-2019#verbosity-settings)):
+
+​                                     | `q[uiet]` | `m[inimal]` | `n[ormal]` | `d[etailed]` | `diag[nostic]`
+------------------------------------- | --------- | ----------- | ---------- | ----------- | --------------
+Errors                                | ✔️        | ✔️         | ✔️         | ✔️         | ✔️   
+Warnings                              | ✔️        | ✔️         | ✔️         | ✔️         | ✔️   
+High-importance Messages              | ❌        | ✔️         | ✔️         | ✔️         | ✔️   
+Normal-importance Messages            | ❌        | ❌         | ✔️         | ✔️         | ✔️   
+Low-importance Messages               | ❌        | ❌         | ❌         | ✔️         | ✔️   
+Additional MSBuild-engine information | ❌        | ❌         | ❌         | ❌         | ✔️   
+
+In other words, you can think of the following log concepts per row as equivalent.
+
+NuGet log level    | MSBuild verbosity switch | MSBuild message type
+------------------ | ------------------------ | --------------------
+`Error`, `Warning` | `q[uiet]`                | Errors and Warnings
+`Minimal`          | `m[inimal]`              | High-importance Messages
+`Information`      | `n[ormal]`               | Normal-importance Messages
+`Verbose`, `Debug` | `d[etailed]`             | Low-importance Messages
+​                   | `diag[nostic]`           | Additional MSBuild-engine information
+
+### Example output
+
+This is output of the invocation of `dotnet nuget verify nuget.common.5.9.0-preview.2.nupkg`
+
+#### Default verbosity | `--verbosity mimimal`
+
+```
+Verifying NuGet.Common.5.9.0-preview.2
+Signature type: Author
+Verifying the author primary signature with certificate:
+  SHA256 hash: 3F9001EA83C560D712C24CF213C3D312CB3BFF51EE89435D3430BD06B5D0EECE
+  Valid from: 2/25/2018 4:00:00 PM to 1/27/2021 4:00:00 AM
+Timestamp: 11/17/2020 7:23:10 AM
```

Comment:
It's not clear what this timestamp is. What do you think of this?

```suggestion
Author primary signature's timestamp: 11/17/2020 7:23:10 AM
```

Reply:
Alternatively, you could separate out the sections. See: https://github.com/NuGet/Home/pull/10466/files#r564105499

Reply:
I like the idea of separating out the sections as it improves readability.